deployer/src/main/java/brooklyn/entity/openshift/OSPaasEntityOpenShiftDriver.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package brooklyn.entity.openshift;
+
+import brooklyn.location.openshift.OpenShiftPaasLocation;
+import com.openshift.client.IOpenShiftConnection;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public abstract class OSPaasEntityOpenShiftDriver implements OSPaasEntityDriver {
```

Comment:
Following the same convention for CloudFoundry entities, I think this should be named "PaasEntityOpenShiftDriver", without the "OS" prefix.